styles/status.cafe/catppuccin.user.css

```
@@ -0,0 +1,125 @@
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+
+@var checkbox colors "Accent color links" 1
+==/UserStyle== */
+
+@-moz-document regexp("https://status\\.cafe/(?!users).*") {
+    @media (prefers-color-scheme: light) {
+        :root {
+            #catppuccin(@lightFlavor, @accentColor);
+        }
+    }
+    @media (prefers-color-scheme: dark) {
+        :root {
+            #catppuccin(@darkFlavor, @accentColor);
+        }
+    }
+
+    #catppuccin(@lookup, @accent) {
+        @rosewater: @catppuccin[@@lookup][@rosewater];
+        @flamingo: @catppuccin[@@lookup][@flamingo];
+        @pink: @catppuccin[@@lookup][@pink];
+        @mauve: @catppuccin[@@lookup][@mauve];
+        @red: @catppuccin[@@lookup][@red];
+        @maroon: @catppuccin[@@lookup][@maroon];
+        @peach: @catppuccin[@@lookup][@peach];
+        @yellow: @catppuccin[@@lookup][@yellow];
+        @green: @catppuccin[@@lookup][@green];
+        @teal: @catppuccin[@@lookup][@teal];
+        @sky: @catppuccin[@@lookup][@sky];
+        @sapphire: @catppuccin[@@lookup][@sapphire];
+        @blue: @catppuccin[@@lookup][@blue];
+        @lavender: @catppuccin[@@lookup][@lavender];
+        @text: @catppuccin[@@lookup][@text];
+        @subtext1: @catppuccin[@@lookup][@subtext1];
+        @subtext0: @catppuccin[@@lookup][@subtext0];
+        @overlay2: @catppuccin[@@lookup][@overlay2];
+        @overlay1: @catppuccin[@@lookup][@overlay1];
+        @overlay0: @catppuccin[@@lookup][@overlay0];
+        @surface2: @catppuccin[@@lookup][@surface2];
+        @surface1: @catppuccin[@@lookup][@surface1];
+        @surface0: @catppuccin[@@lookup][@surface0];
+        @base: @catppuccin[@@lookup][@base];
+        @mantle: @catppuccin[@@lookup][@mantle];
+        @crust: @catppuccin[@@lookup][@crust];
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        color-scheme: if(@lookup = latte, light, dark);
+
+        ::selection {
+            background-color: fade(@accent-color, 30%);
+        }
+
+        input,
+            textarea {
+                &::placeholder {
+                color: @subtext0 !important;
+            }
+        }
+
+        background-color: @base;
+
+        body {
+            background-color: @base;
+            color: @text;
+        }
+        
+        textarea {
+            background-color: @mantle;
+            border-color: @overlay2;
+        }
+        
+        input {
+            background-color: @crust;
+            border-color: @overlay2;
```

```suggestion
            border-color: @overlay2;
            border-style: solid;
            border-width: 1px;
```
Since there is no base input styling (just the browser defaults), setting the border color messes up the appearance. We can set these to make it look normal.